### PR TITLE
Add edpm baremetal CI job

### PR DESCRIPTION
**Pull request by Sandeepyadav93**
With this commit, we add edpm baremetal CI job against dataplane operator.

### CODE REVIEW

**Comment by Sandeepyadav93** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Review by rebtoor — APPROVED** [review]

**Comment by rebtoor** [comment]
recheck

**Comment by Sandeepyadav93** [comment]
recheck

File: zuul.d/projects.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@
     github-check:
       jobs:
         - dataplane-operator-content-provider
-        - dataplane-operator-crc-podified-edpm-deployment:
+        - dataplane-operator-cifmw-crc-podified-edpm-baremetal: &content_provider
```
Comment (Sandeepyadav93) [inline zuul.d/projects.yaml:7]: lets remove cifmw from job name to be consistent with other job name

**Review by fao89 — APPROVED** [review]

**Comment by Sandeepyadav93** [comment]
recheck

**Pull request merged**